contract-examples/.gitignore

```diff
@@ -0,0 +1,144 @@
+/dist
```

Comment:
Should we move this to the repo's own gitignore or keep in this subdirectory?

Reply:
I think it works within directories. and it's probably best to treat this one as a different folder (other than go packages)